contracts/samples/utils/OracleHelper.sol

```diff
@@ -39,7 +39,7 @@ abstract contract OracleHelper {
         /// @notice 'false' if price is dollars-per-ether, 'true' if price is ether-per-dollar
         bool nativeOracleReverse;
 
-        /// @notice The price update threshold percentage that triggers a price update (1e6 = 100%)
+        /// @notice The price update threshold percentage from PRICE_DENOMINATOR that triggers a price update (1e3 = 100%)
```

Comment:
I think `1e3` is better written as `1000`

Reply:
Fixed